servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -267,13 +266,11 @@ public String toString() {
 
         private static final Duration FULL_JITTER = ofDays(1024);
 
-        /**
-         * Special {@link BackOffPolicy} to signal no retries.
-         */
-        public static final BackOffPolicy NO_RETRIES = ofNoRetries();
+        static final BackOffPolicy NO_RETRIES = new BackOffPolicy(0);
```

Comment:
While you are here, I think we should also make a little change for `HttpResponseException`. It currently exposes `metaData` & `message` as public fields instead of using getters. Check out what I did in my other PR (planned to extract it later): https://github.com/apple/servicetalk/pull/2037/files#diff-23091531026e61ea4e2ad39ca2c5aa5280d633b3b728ea06eaa0867474a86409R247-R288